src/main/java/randoop/generation/ForwardGenerator.java

```diff
@@ -93,6 +94,15 @@ public ForwardGenerator(
     this.instantiator = componentManager.getTypeInstantiator();
 
     initializeRuntimePrimitivesSeen();
+
+    if (GenInputsAbstract.small_tests) {
+      inputSequenceSelector = new SmallTestsSequenceSelection();
+    } else if (GenInputsAbstract.enable_orienteering) {
```

Comment:
It would be incorrect for a user to supply both `--small-tests` and `--enable-orienteering`, right.
Randoop should check for this situation.
Please edit `GenInputsAbstract.checkOptionsValid` to do so.

Reply:
Now addressed in GenInputsAbstract.checkOptionsValid.